models/prostate_mri_anatomy/configs/metadata.json

```diff
@@ -1,7 +1,8 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
-    "version": "0.3.2",
+    "version": "0.3.3",
     "changelog": {
+        "0.3.3": "add invertd transformation",
```

Comment:
there's a problem using itk==5.2.1 https://github.com/Project-MONAI/model-zoo/pull/482/checks I think we can try to change the requirement to `"5.2"` or newer @yiheng-wang-nv